catalog/ui/catalog-ui-search/src/main/webapp/component/search-form/search-form.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ /*global require*/
+ var _ = require('underscore');
+ var Backbone = require('backbone');
+
+ module.exports = Backbone.Model.extend({
+    defaults: {
+        name: "A Search Form",
+        createdBy: "admin",
+        createdOn: "30 October 2018",
+        type: "custom",
+        formTemplate: "{}"
```

Comment:
Is the actual template going to be returned, or is it going to be an id that lets us know how to look up the template when the user wants to use it?

Reply:
The current implementation is using a `formTemplate` that has been returned from the backend.